scripts/smeshr/MODIS-STC/MODIS_albedo_basin.sh

```diff
@@ -44,16 +44,16 @@ export PARALLEL_call="parallel --tagstring {#} --tag --line-buffer --jobs ${SLUR
 modis_basin() {
   BASIN_TMP_CUBIC=${1/\.tif/_cubic_tmp.vrt}
   BASIN_TMP_NN=${1/\.tif/_nn_tmp.vrt}
-  BASIN_TMP_MAX=${1/\.tif/_tmp_max.vrt}
+  BASIN_TMP_MAX=${1/\.tif/_tmp_max.tif}
```

Comment:
Curious to hear why the vrt file did not work?

Reply:
My hunch right now is that BASIN_DOMAIN was getting wiped out from line 50 and warping to a vrt with that improper specification threw an error while warping to a tif did not. I think it'll be best to revert to a vrt -  I'll continue testing with other basins today to confirm and push a fix once I get a clean run from scratch

Reply:
Nope, missing BASIN_DOMAIN was one issue, but the [underlying problem](https://gis.stackexchange.com/questions/212288/can-gdal-calc-output-vrts) was that we were attempting to write out a vrt using `gdal_calc.py`. This issue was created when we added the minimum filter step, requiring the BASIN_TMP_MAX variable. Fixed now in [b2b4efd](https://github.com/UofU-Cryosphere/isnoda/pull/64/commits/b2b4efdcf81770adee4b1a9f3f6541e9677c3135)